qiling/arch/cortex_m_const.py

```diff
@@ -31,6 +31,7 @@
     "apsr":      UC_ARM_REG_APSR,
     "ipsr":      UC_ARM_REG_IPSR,
     "epsr":      UC_ARM_REG_EPSR,
+    "cpsr":      UC_ARM_REG_CPSR,
```

Comment:
That is not a good idea... If you decide to go with a temporary solution, then we should do this in a different way. I suggest to do that by modifying render_arm.py line 34:
```python
# there is no cpsr in cortex m
if 'cpsr' in self.ql.arch.regs.register_mapping:
    self.print_mode_info(self.ql.arch.regs.cpsr)
```